content/response-to-reviewers.md

```diff
@@ -218,16 +272,33 @@ Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/69>
+We have added this limitation and our effort to alleviate this issue in the Discussion section of the revised manuscript:
+
+"Because wru was trained on the US census to make predictions, many of the missing predictions are on names not observed in the US census.
+Although the underestimation of the proportion of these names could not be compared between the list of authors and honorees, we complemented this race/ethnicity prediction method with an additional name origin analysis."
+
+Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/70>
 
 > There are assumptions and analysis issues. Did the author check if the association between predicted race/ethnicity and author Affiliation?  Was Asian bias by country or ethnicity?  For example, are American-Asians represented in ISCB honorees but Asians who are geographically located in Asia under-represented. Is this associated with geographic attendance at ISCB meetings? One might expect local awardees in the host country. For example, the authors note that ISMB keynotes had more probability attributable to Israel, while RECOMB had more attributable to East Asian countries.
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/71>
+We agree with the reviewer that the names of the groups of countries are not appropriate.
+In this revision, we have selected more appropriate names for the country grouping to avoid confusion.
+Importantly, to complement the existing analysis, we performed an affiliation analysis to detect countries of affiliation for authors and honorees.
+This is a major improvement of the study because, while examining a different component (affiliation vs. name origin and race/ethnicity), we were able to alleviate biases in the prediction models and analyze the composition of author/honoree affiliation at the country level.
+We also limit our race/ethnicity analysis to only US-affiliated scientists to avoid further biases.
+
+Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/71>.
 
 > Given this small pool of 411, some names of which were duplicated (fellows and keynotes etc), were they manually checked for accuracy.
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/72>
+We did not manually check names and photos to guess gender or name origins to avoid inducing bias.
+Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/72>.
 
 > There are non-random missing data. European Spanish/Portuguese/Italian are called Hispanic. Assignment of the race (white/black/Asian) based surname is difficult to extrapolate to a multi-cultural society where last name and skin color are maybe discordant.
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/73>
+We acknowledged that our supervised learning approaches are neither error free nor bias free.
```

Comment:
```suggestion
We acknowledged that our supervised learning approaches are neither error free nor bias free.
We would also note that it would be a mistake to consider race equivalent to skin color: race, as analyzed in this study, is based on self-identification and is a complex topic with its own area of research.
```
Not sure that this is quite the right wording but I feel we should push back a bit on this skin color == race perspective that's coming through here.